src/test/java/net/openhft/chronicle/queue/impl/single/AppenderFileHandleLeakTest.java

```diff
@@ -61,7 +61,7 @@ public final class AppenderFileHandleLeakTest extends QueueTestCommon {
     private static final SystemTimeProvider SYSTEM_TIME_PROVIDER = SystemTimeProvider.INSTANCE;
     private static final RollCycle ROLL_CYCLE = TEST_SECONDLY;
     private static final DateTimeFormatter ROLL_CYCLE_FORMATTER = DateTimeFormatter.ofPattern(ROLL_CYCLE.format()).withZone(ZoneId.of("UTC"));
-    private static final int TRIES = 5;
+    private static final int TRIES = 100;
```

Comment:
Wow, that may seriously slow down the build.

Reply:
Hi Yevgen, it is used here:

```java
        for (int i = 0; i < TRIES; i++) {
            GcControls.waitForGcCycle();
            openQueueFiles = MappedFileUtil.getAllMappedFiles().stream()
                    .filter(str -> str.contains(queuePath.getAbsolutePath()))
                    .collect(Collectors.toList());
            if (openQueueFiles.isEmpty())
                return true;
            Jvm.pause(10);
        }
```
It should slow it down to worst case 100 * 10 -> 1000ms -> 1s

Reply:
Actually you're right to call this out. `waitForGcCycle will also wait for up to a second... Let me revise the try count down.

Reply:
Going from 5->10 seems to bring the tests relatively stable on Linux for now. Let's go with the 5 over the 100. Thanks for flagging.